azure-kusto-data/tests/test_helpers.py

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License
+import unittest
+import json
+import pytest
+
+from azure.kusto.data.helpers import dataframe_from_result_table
+from azure.kusto.data.response import KustoResponseDataSetV2
+
+
+PANDAS = False
+try:
+    import pandas
+    import numpy
+
+    PANDAS = True
+except:
+    pass
+
+
+# Sample response containing conversion edge cases.
+RESPONSE_TEXT = """
```

Comment:
Move this to a json file like we do in other tests

Reply:
Done!